### PR TITLE
Add testdox phpdoc tag in product reviews page tests

### PR DESCRIPTION
**Pull request by unfulvio-godaddy**
## Summary

Adds `@testdox` phpdoc tags to tests added by the Product Reviews page feature.

## QA

- [x] Code review
- [x] Unit tests pass

### CODE REVIEW

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsTest.php [hunk 1]
```diff
@@ -168,7 +168,7 @@ public function data_provider_get_pending_count_bubble() : Generator {
 	}
 
 	/**
-	 * Tests that it will override the parent file and the submenu file globals when editing a review.
+	 * @testdox The product reviews menu item will be highlighted when editing a review.
```
Comment (agibson-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsTest.php]:
```suggestion
	 * @testdox `edit_review_parent_file` will highlight the product reviews menu item when editing a review.
```

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsTest.php [hunk 2]
```diff
@@ -192,7 +192,7 @@ public function test_edit_review_parent_file() : void {
 	}
 
 	/**
-	 * Tests that it will override the headline text when editing or moderating a review or reply.
+	 * @testdox When editing or moderating a review or a reply to a review the page heading will be updated.
```
Comment (agibson-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsTest.php]:
```suggestion
	 * @testdox `edit_comments_screen_text` will update the page heading when editing or moderating a review or a reply to a review.
```

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsTest.php [hunk 3]
```diff
@@ -331,7 +332,7 @@ public function provider_is_reviews_page() : Generator {
 	}
 
 	/**
-	 * Tests that the admin notice messages are properly returned.
+	 * @testdox After a product review bulk action is processed, the appropriate admin notice is displayed.
```
Comment (agibson-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsTest.php]:
```suggestion
	 * @testdox `get_bulk_action_notice_messages` the appropriate admin notice is displayed after a product review bulk action is processed.
```

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsTest.php [hunk 4]
```diff
@@ -459,7 +460,7 @@ public function provider_get_bulk_action_notice_messages() : Generator {
 	}
 
 	/**
-	 * Tests that a notice message will result in a valid HTML return.
+	 * @testdox When a product reviews bulk action notice is to be displayed, it output the appropriate message HTML.
```
Comment (agibson-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsTest.php]:
```suggestion
	 * @testdox `maybe_display_reviews_bulk_action_notice` will output the appropriate message HTML for a product reviews bulk action notice.
```

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsTest.php [hunk 5]
```diff
@@ -41,7 +41,7 @@ public function tearDown() : void {
 	}
 
 	/**
-	 * Tests that can get the capability to view the reviews page.
+	 * @testdox Gets the filterable user capability for viewing the product reviews page.
```
Comment (agibson-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsTest.php]:
```suggestion
	 * @testdox `get_capability` gets the filterable user capability for viewing the product reviews page.
```

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsTest.php [hunk 6]
```diff
@@ -582,7 +583,7 @@ public function provider_is_review_or_reply(): Generator {
 	}
 
 	/**
-	 * Tests different cases that require factories.
+	 * @testdox A comment object is considered a review or a reply if the `comment_type` is `comment` or `review` and the parent post is a product.
```
Comment (agibson-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsTest.php]:
```suggestion
	 * @testdox `is_review_or_reply` correctly determines if an object is a review or a reply to a review.
```

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsUtilTest.php [hunk 7]
```diff
@@ -36,7 +36,7 @@ public function tearDown() : void {
 	}
 
 	/**
-	 * Tests that can get the comment clauses excluding product reviews.
+	 * @testdox Comment clauses will be returned excluding product reviews if the current screen is for the `edit-comments` page.
```
Comment (agibson-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsUtilTest.php]:
```suggestion
	 * @testdox `comments_clauses_without_product_reviews` modifies the comment query clauses to exclude product reviews if the current screen is for the `edit-comments` page.
```

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php [hunk 8]
```diff
@@ -82,7 +82,7 @@ public function test_single_row() : void {
 	}
 
 	/**
-	 * Tests that can handle the row actions for a review in the Reviews page table.
+	 * @testdox Each row of the product reviews page table will have admin action links pertaining each review or comment.
```
Comment (agibson-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php]:
```suggestion
	 * @testdox `handle_row_actions` displays admin action links pertaining each review or comment.
```

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php [hunk 9]
```diff
@@ -182,7 +182,7 @@ public function test_get_columns() : void {
 	}
 
 	/**
-	 * Tests that can get the product reviews' page columns when a filter is applied.
+	 * @testdox The product reviews page columns can be filtered.
```
Comment (agibson-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php]:
```suggestion
	 * @testdox `get_columns` returns columns for the product reviews page that can be filtered.
```

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php [hunk 10]
```diff
@@ -1125,7 +1129,7 @@ public function provider_get_search_arguments() : Generator {
 	}
 
 	/**
-	 * Tests that can output the text for when no reviews are found.
+	 * @testdox When no reviews are found for a given request a message is returned as the product reviews page table content.
```
Comment (agibson-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php]:
```suggestion
	 * @testdox `no_items` returns custom text when no reviews are found for a given request.
```

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php [hunk 11]
```diff
@@ -1154,7 +1158,7 @@ public function data_provider_no_items() : Generator {
 	}
 
 	/**
-	 * Tests that can render the extra controls for the product reviews page.
+	 * @testdox The product reviews page will offer additional UI elements allowing merchants to filter the reviews to display.
```
Comment (agibson-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php]:
```suggestion
	 * @testdox `extra_tablenav` will offer additional UI elements allowing merchants to filter the reviews to display.
```

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php [hunk 12]
```diff
@@ -1480,7 +1484,7 @@ public function test_column_default() : void {
 	}
 
 	/**
-	 * Tests that it will trigger a filter hook for columns.
+	 * @testdox Each column in the product reviews page table can be filtered by third parties.
```
Comment (agibson-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php]:
```suggestion
	 * @testdox `filter_column_output` allows columns in the product reviews page table to be filtered by third parties.
```

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php [hunk 13]
```diff
@@ -1866,6 +1872,8 @@ public function test_get_per_page() {
 	}
 
 	/**
+	 * @testdox Each product review in the list table will have a column displaying a bubble with information about pending and approved reviews for the corresponding product.
```
Comment (agibson-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php]:
```suggestion
	 * @testdox `comments_bubble` displays a bubble with information about pending and approved reviews for the corresponding product.
```

**Review by agibson-godaddy — CHANGES_REQUESTED** [review]
I'm not really familiar with `testdox`, but in the few examples I looked at in Woo, they were all prefixed with the method name being tested. So the changes I requested are just to add a few method name prefixes that were missing. Not sure if it's totally required or not. 🤷

**Comment by unfulvio-godaddy** [comment]
@agibson-godaddy according to https://phpunit.readthedocs.io/en/9.5/annotations.html#testdox the tag doesn't need to begin with a method name, seems pretty flexible. I'm also unfamiliar with it: I suppose it exists for parsers to read its content and do something with it. Nonetheless A8C might be using these with the convention you pointed out and I've accepted all your suggestions to conform with that. Thanks for reviewing!

**Review by agibson-godaddy — APPROVED** [review]
LGTM!

**Pull request merged**